src/main/resources/db/migration/V1__create-tables.sql

```diff
@@ -1,6 +1,3 @@
---liquibase formatted sql
```

Comment:
use liquibase

Reply:
I'm using r2dbc-migrate tool. It's a bad choice?

Reply:
@eyakauleva prefer to use liquibase, it can be utilized with r2dbc & jdbc connectors